source/Sprite.cpp

```diff
@@ -65,6 +65,16 @@ Sprite::Sprite(const string &name)
 
 
 
+Sprite::~Sprite()
+{
+	if(texture[0]) glDeleteTextures(1, &texture[0]);
+	if(texture[1]) glDeleteTextures(1, &texture[1]);
+	if(swizzleMask[0]) glDeleteTextures(1, &swizzleMask[0]);
+	if(swizzleMask[1]) glDeleteTextures(1, &swizzleMask[1]);
```

Comment:
Does adding this cause a delay when closing the game? Presumably it will? If so, we might want to manually unload the sprites instead of relying on the destructor.

Is this the case for sounds as well?

Reply:
I tried it out with high dpi and some other large plugins, and didn't really feel the difference. I didn't have a bunch of custom sounds to try it out with, though.

Reply:
you should not notice any difference because either you do it here or the OS does it by itself at program end, will be the same result. only difference would be the checks, but I dont think that will make a large difference